recipe/meta.yaml

```diff
@@ -339,9 +344,8 @@ outputs:
         # https://github.com/conda-forge/conda-forge.github.io/issues/1528
         - openssl
         - {{ pin_subpackage('tensorflow-base', exact=True) }}
-        # Keep the cuda version here since it helps package solvers
-        # decide on the cuda variant
-        - cuda-version {{ cuda_compiler_version }}  # [cuda_compiler_version != "None"]
+        # avoid that people without GPUs needlessly download ~200-300MB
+        - __cuda  # [cuda_compiler_version != "None"]
```

Comment:
Shouldn't this go into run?

Reply:
yes.

Reply:
the change is unecessary since everything requires libtensorflow exact.